Meridian59/Data/Models/AdminData/AdminData.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Meridian59.Common.Interfaces;
+using Meridian59.Data.Lists;
+using Meridian59.Protocol.Events;
+using Meridian59.Protocol.GameMessages;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    public class AdminData : INotifyPropertyChanged, IClearable
+    {
+        //Handles the objects we're watching in the AdminUI
+        private BaseList<AdminObject> TrackedObjects { get; set; }
```

Comment:
Have a look at the comments somewhere else about creating instances of internal collections exactly once at class initialisation by using the 'readonly' modifier (I know I haven't updated all my own code to this yet, but it's the way to go).

>> protected readonly BaseList< AdminObject > trackedObjects = new BaseList< AdminObject >();
>> public BaseList< AdminObject > TrackedObjects { get { return trackedObjects; } }

I think this should rather be